src/styles/index.scss

```diff
@@ -2,2 +1,38 @@
+@use './blocks/header.scss';
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+@use './utils/variables' as *;
+
+@mixin column-count($count) {
+  grid-template-columns: repeat($count, $card-width);
+}
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  box-sizing: border-box;
+  color: $black-color;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+
+  @include column-count(1);
```

Comment:
The `column-count` mixin is used here. Ensure that this is the intended method for setting grid columns to avoid redundancy with the direct `grid-template-columns` definition.